src/main/java/com/powsybl/openloadflow/OpenLoadFlowProvider.java

```diff
@@ -266,7 +267,7 @@ private LoadFlowResult runDc(Network network, LoadFlowParameters parameters, Ope
                                                                      true,
                                                                      parameters.getConnectedComponentMode() == LoadFlowParameters.ConnectedComponentMode.MAIN);
 
-        List<DcLoadFlowResult> results = new DcLoadFlowEngine(network, dcParameters, reporter)
+        List<DcLoadFlowResult> results = new DcLoadFlowEngine(network, dcParameters, reporter, parameters.getCountriesToBalance())
```

Comment:
Because I think it should neither be in AcParameters nor in DcParameters. I modified it to have the symmetry but I think the parameters class need to be refactorized to avoid duplication.